### DISCUSSION TITLE
6월 21일 논의 할 내용

### DISCUSSION THREAD

**Discussion by gracefulBrown** [woowacourse/prolog#122] [Notice]
베타 서비스 오픈 하기 전 까지 필요한 기능 목록을 모아서 우선순위를 정할 예정인데요!
[서비스](https://dev.pro-log.dev) 사용 후 코멘트를 남겨주세요!
남겨준 코멘트를 기반으로 오늘 회의에서 향후 일정을 계획하겠습니다

## 코멘트에 필요한내용
- 베타 오픈 시 까지 반드시 필요한 기능 목록 (우선순위 순서로)
- 베타 오픈 때 포함되면 좋을 것 같은 기능 목록 (순서 없이)

**회의는 오후 3시로 계획하겠습니다!(변경 가능!!)**

**Comment by gracefulBrown** [comment]
## 반드시 필요한 기능
1. 로그아웃 기능
2. 미션, 필터 목록 노출 방법 변경(현재 미션 목록이 너무 많아서 보기 불편함)
3. 마이페이지 -> 내가 쓴 글 목록 페이지로 변경
4. 내가 쓴 글 목록 페이지 > 수정 / 삭제 기능

## 포함하고 싶은 기능
- 글 작성 시간 표시 방법 변경
- 유저별 글 목록 조회 기능

**Reply by sihyung92** [reply]
# 반.필.기
1. 글 목록 순서가 역 최신순임 (노출 순서 조정)
2. 상세화면 markdown 적용
3. 글 내용 수정
4. 필터를 추가할 때 개인적으로 #을 붙이는 버릇이 있는데, 첫 글자가 #이면 #은 제외하는 기능
# 포.싶.기
1. 유저를 클릭하면 해당 유저의 프로필이나 모음글을 볼 수 있는 기능

**Comment by sunhpark42** [comment]
### 반드시 필요한 기능
1. 현재 유저 이름이 없음.(버그픽스 필요)
2. 마이페이지 필요하다 : 어떤 글을 수정하고 삭제할 수 있게 할 것인지? 
3. 서버에서 한 번에 글을 몇개씩 보내줄 것인지? 정할 필요가 있을 것 같다. (ex. 한 번에 10개씩)
4. 각 포스트 별 마크다운 적용.
5. 필터 초기화 기능

### 포함하면 좋을 기능
1. 검색기능

**Comment by gracefulBrown** [comment]
# 엘라
반필기
1. 글 수정 기능
2. 글 작성 > 스타일 적용(폰트 사이즈 등)
3. 메인 다중 필터 기능
4. 필터(미션, 태그 등) 목록 노출

# 소롱 
반필기
1. 글 수정
2. 유저별 필터

추가 사항
1. 마이페이지는 새 페이지가 필요해서 시간이 좀 더 걸리지 않늘까요? 우선 유저별 필터로..

**Comment by gracefulBrown** [comment]
# 반드시 필요한 기능
## 로그아웃 기능
- 로컬스토리지에서 지우고 새로고침
- #113

## 필터 목록 노출 방법
- 스크롤 처리
- 이후 추후 논의
- 입력 부분 가리기
- #129

## 마이페이지(내가 쓴 글 목록), 글 수정/삭제
- 마이페이지 기능 설계 우선
- **추후 논의**
- #128

## 포스트 목록 정렬(순서) & 페이징 
- #124
- ![image](https://user-images.githubusercontent.com/46308949/122716949-e64a5500-d2a5-11eb-82a3-9a64f9f80b42.png)

## 상세화면 markdown 적용
- 조금 걸릴 수 있음.. 
- #125

## 현재 유저 이름이 없음.(버그픽스 필요)
- #123

## 글 작성 > 스타일 적용(폰트 사이즈 등)

# 추가적인 기능
## 유저별 필터
## 필터를 추가할 때 개인적으로 #을 붙이는 버릇이 있는데, 첫 글자가 #이면 #은 제외하는 기능
